bauble/src/value/convert.rs

```diff
@@ -0,0 +1,1301 @@
+use crate::{
```

Comment:
how much was changed in this file or just moved?

Reply:
It was changed quite a lot, because I deduplicated some code. The conversion from `ParseVal` -> `Val`, `CopyVal` -> `Val` and `Val` -> `Val` now all reuse the saem conversion code